### PR TITLE
Fix all LR(1) conflicts

### PR DESCRIPTION
**Pull request by alexrefshauge**
Eliminated differantiation between logical and arithmetic expressions
Removed string from types

### CODE REVIEW

**Comment by alexrefshauge** [comment]
Then we update bnf in report

File: pkg/ast/controlStatementActions.go [hunk 1]
```diff
@@ -1,20 +1,29 @@
...
+
+func NewIfStatement(condition, bodyBlock, elseBlock Attribute) (*IfStatement, error) {
 	var conditionExpression Expression
 	var bodyStatementList StatementList
 	var elseStatementList StatementList
 
 	conditionExpression = condition.(Expression)
 
+	op := string(conditionExpression.Operator.(*token.Token).Lit)
+	if op == "+" || op == "-" || op == "/" || op == "*" {
```
Comment (MathiasFM) [inline pkg/ast/controlStatementActions.go:16]: modulo?

Comment (alexrefshauge) [inline pkg/ast/controlStatementActions.go:16]:
```suggestion
	if op == "+" || op == "-" || op == "/" || op == "*" || op == "%" {
```

Comment (alexrefshauge) [reply pkg/ast/controlStatementActions.go]: donzo

**Review by MathiasFM — APPROVED** [review]

**Review by GustavHus — APPROVED** [review]
True LGTM

**Pull request merged**